foxglove/cmd/attachments.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/foxglove/foxglove-cli/foxglove/console"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func newListAttachmentsCommand(params *baseParams) *cobra.Command {
+	var format string
+	var importID string
+	var recordingID string
+	attachmentsListCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List MCAP attachments",
+		Run: func(cmd *cobra.Command, args []string) {
+			client := console.NewRemoteFoxgloveClient(
+				*params.baseURL, *params.clientID,
+				viper.GetString("bearer_token"),
+				params.userAgent,
+			)
+			err := renderList(
+				os.Stdout,
+				&console.AttachmentsRequest{
+					ImportID:    importID,
+					RecordingID: recordingID,
+				},
+				client.Attachments,
+				format,
+			)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Failed to list imports: %s\n", err)
+				os.Exit(1)
+			}
+		},
+	}
+	attachmentsListCmd.InheritedFlags()
+	attachmentsListCmd.PersistentFlags().StringVarP(&importID, "import-id", "", "", "Import ID")
+	attachmentsListCmd.PersistentFlags().StringVarP(&recordingID, "recording-id", "", "", "Recording ID")
+	AddFormatFlag(attachmentsListCmd, &format)
+	return attachmentsListCmd
+}
+
+func newGetAttachmentCmd(params *baseParams) *cobra.Command {
+	attachmentsGetCmd := &cobra.Command{
+		Use:   "get",
```

Comment:
To me, "get" feels like it should correspond to application/json representations of an API resource. Since this is at `/download`, I might expect "download", "export", or something like that.

As a user, "get" doesn't feel wrong, but if we end up with other commands which all correspond to `/resource/:id` routes, and let me switch the output format, this inconsistency could end up sticking out.

Reply:
yeah, I can change to download